src/app/pages/onboarding/sign-in/hooks/use-sign-in.ts

```diff
@@ -58,6 +59,7 @@ export function useSignIn() {
 
       await simulateShortDelayToAvoidImmediateNavigation();
 
+      dispatch(keyActions.signOut());
```

Comment:
need to remove current software keys, otherwise user can not set new pwd form and go directly to home page